src/include/rbd/librbd.h

```diff
@@ -733,6 +733,8 @@ CEPH_RBD_API int rbd_snap_get_timestamp(rbd_image_t image, uint64_t snap_id, str
 
 CEPH_RBD_API int rbd_snap_set(rbd_image_t image, const char *snapname);
 CEPH_RBD_API int rbd_snap_set_by_id(rbd_image_t image, uint64_t snap_id);
+CEPH_RBD_API int rbd_snap_get_name_by_id(rbd_image_t image, char *snapname, uint64_t snap_id, size_t *name_len);
```

Comment:
Nit: the `_by_id` and `_by_name` suffixes are probably unnecessary since there isn't a need to differentiate between the two.